src/main/java/lotto/generator/LottoGenerator.java

```diff
@@ -1,7 +1,45 @@
 package lotto.generator;
 
 import lotto.domain.Lotto;
+import lotto.domain.LottoNumber;
+import lotto.view.OutputView;
 
-public interface LottoGenerator {
-    abstract Lotto create();
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+    private static List<LottoNumber> totalRandomNumbers;
```

Comment:
```suggestion
    private static Map<Integer, LottoNumber> lottoNumbers;
```
LottoNumber에 Map으로 구현하고 
private LottoNumber(int number), private LottoNumber(String number)으로 바꾸고
public static LottoNumber valueOf(String number)를 추가해서 캐시로 활용해보면 어떨까요?